ext/AvizExt/viz/scenarios.jl

```diff
@@ -55,6 +55,25 @@ function ADRIA.viz.scenarios(
         series_opts=series_opts,
     )
 end
+function ADRIA.viz.scenarios!(
```

Comment:
Because the usual use case is:

```julia
s_tac = ADRIA.metrics.scenario_total_cover(rs)
ADRIA.viz.scenarios(rs, s_tac)
```

By that pattern, you would expect adding to an existing figure would follow the same pattern, just providing the figure location to use:

```julia
ADRIA.viz.scenarios!(fig[1,2], rs, s_tac)
```

Without it, it breaks this expectation and thus causes friction.
In fact, I thought we had it - previous version of the code in the documentation was like this and was working so I don't know what happened.